File: Python/Nested_Conditional_Statements(EXERCISE)/Journey.py
budget = float(input())
season = input()
holiday_house = ""
sum = 0
place = ""

if season == "summer":
    holiday_house = "Camp"

    if budget <= 100:
        sum = budget - 30 / 100 * budget
        place = "Bulgaria"
    elif 100 < budget <= 1000:
        sum = budget - 40 / 100 * budget
        place = "Balkans"
    elif 1000 < budget:
        holiday_house = "Hotel"
        sum = budget - 90 / 100 * budget
        place = "Europe"
    budget = budget - sum

    print(f"Somewhere in {place}")
    print(f"{holiday_house} - {budget:.2f}")

if season == "winter":
    holiday_house = "Hotel"

    if budget <= 100:
        sum = budget - 70 / 100 * budget
        place = "Bulgaria"
    elif 100 < budget <= 1000:
        sum = budget - 80 / 100 * budget
        place = "Balkans"
    elif 1000 < budget:
        sum = budget -  90 / 100 * budget
        place = "Europe"
    budget = budget - sum

    print(f"Somewhere in {place}")
    print(f"{holiday_house} - {budget:.2f}")

File: Java Programing Basics/EXERCISE NestedConditionalStatelments/src/Journey.java
import java.util.Scanner;

public class Journey {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double money = Double.parseDouble(scanner.nextLine());
        String season = scanner.nextLine();

        if (money <= 100) {
            if (season.equalsIgnoreCase("Summer")) {
                double spendMoney = (30 / 100.0) * money;
                System.out.println("Somewhere in Bulgaria");
                System.out.printf("Camp - %.2f", spendMoney);
            }

            if (season.equalsIgnoreCase("Winter")) {
                double spendMoney = (70 / 100.0) * money;
                System.out.println("Somewhere in Bulgaria");
                System.out.printf("Hotel - %.2f", spendMoney);
            }
        }

        if (money <= 1000 && money > 100) {
            if (season.equalsIgnoreCase("Summer")) {
                double spendMoney = (40 / 100.0) * money;
                System.out.println("Somewhere in Balkans");
                System.out.printf("Camp - %.2f", spendMoney);
            }

            if (season.equalsIgnoreCase("Winter")) {
                double spendMoney = (80 / 100.0) * money;
                System.out.println("Somewhere in Balkans");
                System.out.printf("Hotel - %.2f", spendMoney);
            }
        }

            if (money > 1000) {
                if (season.equalsIgnoreCase("Summer")) {
                    double spendMoney = (90 / 100.0) * money;
                    System.out.println("Somewhere in Europe");
                    System.out.printf("Hotel - %.2f", spendMoney);
                }

                if (season.equalsIgnoreCase("Winter")) {
                    double spendMoney = (90 / 100.0) * money;
                    System.out.println("Somewhere in Europe");
                    System.out.printf("Hotel - %.2f", spendMoney);
                }


            }


    }
}